frontend-service/build.gradle

```diff
@@ -60,15 +60,8 @@ task swagger(type: GenerateSwaggerModel, dependsOn: classes) {
     scanClasspath = project.sourceSets.main.runtimeClasspath
 
     // TODO: Specify API classes by package rather than name
```

Comment:
You can probably remove this TODO now ;)

Reply:
Will do